spec/acceptance/php56_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper_acceptance'
+
+describe 'with specific php version' do
+  case default[:platform]
+  when %r{16.04}
```

Comment:
Wouldn't it be better to just check for Ubuntu here?

Reply:
yeah would work as well